### PR TITLE
Added rental entity

### PR DESCRIPTION
**Pull request by KyrylH**

### CODE REVIEW

File: src/main/java/com/example/carsharingservice/model/Rental.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    private Long id;
+    @Column(name = "rental_start_date")
+    private LocalDate rentalStart;
+    @Column(name = "rental_return_date")
+    private LocalDate rentalReturn;
+    @Column(name = "actual_return_date")
+    private LocalDate actualRentalReturn;
+    @ManyToOne(fetch = FetchType.LAZY)
```
Comment (OlehSarapuk) [inline src/main/java/com/example/carsharingservice/model/Rental.java:34]: @JoinColumn with name car_id

File: src/main/java/com/example/carsharingservice/model/Rental.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    private Car car;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User user;
```
Comment (OlehSarapuk) [inline src/main/java/com/example/carsharingservice/model/Rental.java:41]: @joincolumn with name user_id

File: src/main/java/com/example/carsharingservice/model/Rental.java (hunk 1 above)
Comment (OlehSarapuk) [reply src/main/java/com/example/carsharingservice/model/Rental.java:34]: @ToString.Exclude
    @EqualsAndHashCode.Exclude

File: src/main/java/com/example/carsharingservice/model/Rental.java (hunk 2 above)
Comment (OlehSarapuk) [reply src/main/java/com/example/carsharingservice/model/Rental.java:41]: @ToString.Exclude
    @EqualsAndHashCode.Exclude

**Review by OlehSarapuk — CHANGES_REQUESTED** [review]

**Review by IvanZuravlev228 — APPROVED** [review]

**Review by OlehSarapuk — APPROVED** [review]

**Pull request merged**